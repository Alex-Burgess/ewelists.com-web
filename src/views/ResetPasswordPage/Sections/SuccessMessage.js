import React from 'react';
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles";
// core components
import HeaderTransparent from "custom/Header/HeaderTransparent.js";
import FooterTransparent from "custom/Footer/FooterTransparent.js";
import GridContainer from "components/Grid/GridContainer.js";
import GridItem from "components/Grid/GridItem.js";
import Card from "components/Card/Card.js";
import CardBody from "components/Card/CardBody.js";
import config from 'config.js';

import styles from "assets/jss/custom/views/resetPasswordPageStyle.js";
const useStyles = makeStyles(styles);

const image = config.imagePrefix + "/images/sheep-with-shoes.jpg";

export default function SuccessMessage(props) {
  const classes = useStyles();

  return (
    <div>
      <HeaderTransparent isAuthenticated={false} />
      <div
        className={classes.pageHeader}
        style={{
          backgroundImage: "url(" + image + ")",
          backgroundSize: "cover",
          backgroundPosition: "top center"
        }}
      >
        <div className={classes.container}>
          <GridContainer justify="center">
            <GridItem xs={12} sm={12} md={5}>
              <Card className={classes.cardSignup}>
                <h2 className={classes.cardTitle}>Password Reset Complete</h2>
                <CardBody className={classes.textCenter}>
                  <div id="successMessage" className={classes.details}>
                    <p>Your password has been reset.</p>
                    <p>
                      <a href="/login" className={classes.link}>Login with your new credentials</a>
                    </p>
                  </div>

                </CardBody>
              </Card>
            </GridItem>
          </GridContainer>
        </div>
        <FooterTransparent />
      </div>
    </div>
  );
}
