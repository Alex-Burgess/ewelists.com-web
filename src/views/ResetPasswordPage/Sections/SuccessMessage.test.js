import * as React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import SuccessMessage from './SuccessMessage';

describe('Reset Password Success Message Snapshots', () => {
  test('Renders success message', () => {
    const tree = TestRenderer
      .create(<Router><SuccessMessage /></Router>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
