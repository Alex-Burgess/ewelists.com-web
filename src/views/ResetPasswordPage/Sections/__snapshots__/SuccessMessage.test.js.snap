// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset Password Success Message Snapshots Renders success message 1`] = `
<div>
  <div>
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed makeStyles-appBar-70 makeStyles-transparent-82 makeStyles-fixed-72 MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular makeStyles-container-73 MuiToolbar-gutters"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-title-74"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            <a
              href="/"
              onClick={[Function]}
            >
              <img
                alt="logo"
                className="makeStyles-logo-69"
                src="logo-white.png"
              />
            </a>
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
        <div
          className="makeStyles-hidden-86 PrivateHiddenCss-smDown-168"
        >
          <div
            className="makeStyles-collapse-87"
          >
            <ul
              className="MuiList-root makeStyles-list-178 makeStyles-mlAuto-194 MuiList-padding"
            >
              <li
                className="MuiListItem-root makeStyles-listItem-179 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-navLink-181"
                  href="/listideas"
                  onClick={[Function]}
                >
                  Gift List Ideas
                </a>
              </li>
              <li
                className="MuiListItem-root makeStyles-listItem-179 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-navLink-181"
                  href="/login"
                  onClick={[Function]}
                >
                  Log In
                </a>
              </li>
              <li
                className="MuiListItem-root makeStyles-listItem-179 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-navLink-181"
                  href="/signup"
                  onClick={[Function]}
                >
                  Sign Up
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </header>
  </div>
  <div
    className="makeStyles-pageHeader-2"
    style={
      Object {
        "backgroundImage": "url(sheep-with-shoes.jpg)",
        "backgroundPosition": "top center",
        "backgroundSize": "cover",
      }
    }
  >
    <div
      className="makeStyles-container-1"
    >
      <div
        className="MuiGrid-root makeStyles-grid-210  MuiGrid-container MuiGrid-justify-xs-center"
      >
        <div
          className="MuiGrid-root makeStyles-grid-314  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-5"
        >
          <div
            className="makeStyles-card-315 makeStyles-cardSignup-3"
          >
            <h2
              className="makeStyles-cardTitle-4"
            >
              Password Reset Complete
            </h2>
            <div
              className="makeStyles-cardBody-330 makeStyles-textCenter-32"
            >
              <div
                className="makeStyles-details-45"
                id="successMessage"
              >
                <p>
                  Your password has been reset.
                </p>
                <p>
                  <a
                    className="makeStyles-link-48"
                    href="/login"
                  >
                    Login with your new credentials
                  </a>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      className="makeStyles-footer-349 makeStyles-footer-343"
    >
      <div
        className="makeStyles-container-355"
      >
         
        <div>
          <div
            className="makeStyles-left-340"
          >
            <ul
              className="MuiList-root makeStyles-list-339 MuiList-padding"
            >
              <li
                className="MuiListItem-root makeStyles-inlineBlock-338 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-block-337"
                  href="/"
                >
                  Â© 
                  2020
                   ewelists
                </a>
              </li>
              <li
                className="MuiListItem-root makeStyles-inlineBlock-338 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-block-337"
                  href="/contact"
                >
                  Contact Us
                </a>
              </li>
              <li
                className="MuiListItem-root makeStyles-inlineBlock-338 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-block-337"
                  href="/#about"
                >
                  About us
                </a>
              </li>
              <li
                className="MuiListItem-root makeStyles-inlineBlock-338 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-block-337"
                  href="/privacy"
                >
                  Privacy
                </a>
              </li>
              <li
                className="MuiListItem-root makeStyles-inlineBlock-338 MuiListItem-gutters"
                disabled={false}
              >
                <a
                  className="makeStyles-block-337"
                  href="/terms"
                >
                  Terms
                </a>
              </li>
            </ul>
          </div>
          <div
            className="makeStyles-rightLinks-345"
          >
            <ul>
              <li>
                <a
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-366 makeStyles-simple-388 makeStyles-justIcon-396"
                  href="https://www.facebook.com/ewelists"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={0}
                  target="_blank"
                >
                  <span
                    className="MuiButton-label"
                  >
                    <i
                      className="fab fa-facebook"
                    />
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </a>
              </li>
              <li>
                <a
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-366 makeStyles-simple-388 makeStyles-justIcon-396"
                  href="https://twitter.com/ewelists"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={0}
                  target="_blank"
                >
                  <span
                    className="MuiButton-label"
                  >
                    <i
                      className="fab fa-twitter"
                    />
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </a>
              </li>
              <li>
                <a
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-366 makeStyles-simple-388 makeStyles-justIcon-396"
                  href="https://www.instagram.com/ewelists/"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={0}
                  target="_blank"
                >
                  <span
                    className="MuiButton-label"
                  >
                    <i
                      className="fab fa-instagram"
                    />
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="makeStyles-clearFix-365"
        />
      </div>
    </footer>
  </div>
</div>
`;
